pkgs/applications/audio/virtual-ans/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv
+, fetchurl
+, unzip
+, xorg
+, libGL
+, alsaLib
+, SDL2
+, autoPatchelfHook
+}:
+
+stdenv.mkDerivation rec {
+  name = "virtual-ans-${version}";
+  version = "3.0.2c";
+
+  src = fetchurl {
```

Comment:
Should be using `fetchzip` here, this way you don't have to manually unzip

Reply:
I'm struggling to understand were `fetchzip` puts the output.

Reply:
> I'm struggling to understand were fetchzip puts the output.

... dumps it all in the build's working directory, apparently ...